### PR TITLE
add SCOPE field content for docker network ls

### PR DESCRIPTION
**Pull request by lixiaobing1**
add SCOPE field content for docker network ls.

### CODE REVIEW

File: man/docker-network-ls.1.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ networks that span across multiple hosts in a cluster, for example:
...
-    NETWORK ID          NAME                DRIVER
-    7fca4eb8c647        bridge              bridge
-    9f904ee27bf5        none                null
-    cf03ee007fb4        host                host
-    78b03ee04fc4        multi-host          overlay
+    NETWORK ID          NAME                DRIVER          SCOPE
+    7fca4eb8c647        bridge              bridge          local
+    9f904ee27bf5        none                null            local
+    cf03ee007fb4        host                host            local
+    78b03ee04fc4        multi-host          overlay         local
```
Comment (mavenugo) [inline man/docker-network-ls.1.md]: Overlay driver must be `swarm` scoped. can u pls fix that ?

**Comment by lixiaobing10051267** [comment]
ok, has been modified, another file network_ls.md also modified.

**Comment by mavenugo** [comment]
thanks. LGTM

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]
LGTM

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**